### PR TITLE
Fix newsletter dialog while registration.

### PR DESCRIPTION
**Pull request by yeryomenkom**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1290,6 +1290,7 @@
 
     <string name="app_entry_dialog_accept">ACCEPT</string>
     <string name="app_entry_dialog_cancel">CANCEL</string>
+    <string name="app_entry_dialog_no_thanks">NO, thanks</string>
```
Comment (JPonte) [inline app/src/main/res/values/strings.xml]: Why isn't `thanks` capitalised?

**Comment by yeryomenkom** [comment]
build again

**Comment by yeryomenkom** [comment]
build again

**Comment by yeryomenkom** [comment]
build again

**Pull request merged**